### PR TITLE
Multitool Examine runtime fix

### LINKED ISSUE #1060: Examining a multitool runtimes
```
[23:28:06] Runtime in mob.dm, line 454: Cannot execute null.Join().
verb name: Examine (/mob/verb/examinate)
usr: Ike709/(Adriana Gettemy)
usr.loc: (Vault (82, 147, 2))
src: Adriana Gettemy (/mob/living/carbon/human)
src.loc: the floor (82,147,2) (/turf/open/floor/circuit)
call stack:
Adriana Gettemy (/mob/living/carbon/human): Examine(the multitool (/obj/item/multitool))
the multitool (/obj/item/multitool): ShiftClick(Adriana Gettemy (/mob/living/carbon/human))
Adriana Gettemy (/mob/living/carbon/human): ShiftClickOn(the multitool (/obj/item/multitool))
Adriana Gettemy (/mob/living/carbon/human): ClickOn(the multitool (/obj/item/multitool), "icon-x=15;icon-y=21;left=1;shi...")
the multitool (/obj/item/multitool): Click(null, "mapwindow.map", "icon-x=15;icon-y=21;left=1;shi...")
Ike709 (/client): Click(the multitool (/obj/item/multitool), null, "mapwindow.map", "icon-x=15;icon-y=21;left=1;shi...")
```

### PR DESCRIPTION
**Pull request by Kerbin-Fiber**
## About The Pull Request
Closes #1060 
## Why It's Good For The Game
fix, duh

## Changelog
:cl:
fix: Multitools no longer runtime on examine.
/:cl:

### CODE REVIEW

**Comment by ike709** [comment]
Actually, `. += [whatever]` is the proper way to do examine() subtypes these days. Please convert the to_chats.

File: code/game/objects/items/devices/multitool.dm [hunk 1]
```diff
@@ -44,11 +40,11 @@
 
 //I know copypasting it and overwriting the old procs is a pretty rough thing to do, but hey, it's the simplest and most effective
 /obj/item/multitool/examine(mob/user)
-	..()
+	. = ..()
 	if(selected_io)
-		to_chat(user, "<span class='notice'>Activate [src] to detach the data wire.</span>")
+		. += [user, "<span class='notice'>Activate [src] to detach the data wire.</span>"]
```
Comment (ike709) [inline code/game/objects/items/devices/multitool.dm]:
```suggestion
		. += "<span class='notice'>Activate [src] to detach the data wire.</span>"
```

File: code/game/objects/items/devices/multitool.dm [hunk 2]
```diff
@@ -44,11 +40,11 @@
...
 //I know copypasting it and overwriting the old procs is a pretty rough thing to do, but hey, it's the simplest and most effective
 /obj/item/multitool/examine(mob/user)
-	..()
+	. = ..()
 	if(selected_io)
-		to_chat(user, "<span class='notice'>Activate [src] to detach the data wire.</span>")
+		. += [user, "<span class='notice'>Activate [src] to detach the data wire.</span>"]
 	if(buffer)
-		to_chat(user, "<span class='notice'>Its buffer contains [buffer].</span>")
+		. += [user, "<span class='notice'>Its buffer contains [buffer].</span>"]
```
Comment (ike709) [inline code/game/objects/items/devices/multitool.dm]:
```suggestion
		. += "<span class='notice'>Its buffer contains [buffer].</span>"
```

**Comment by Kerbin-Fiber** [comment]
Oh ok, so that's how it's done

**Review by qwertyquerty — APPROVED** [review]

**Pull request merged**